core/connector/src/main/java/org/exoplatform/wcm/connector/FileUploadHandler.java

```diff
@@ -304,6 +304,7 @@ public Response checkExistence(Node parent, String fileName) throws Exception {
     DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
     DocumentBuilder builder = factory.newDocumentBuilder();
     Document fileExistence = builder.newDocument();
+    fileName = Utils.cleanNameWithAccents(fileName);
     fileName = Text.escapeIllegalJcrChars(fileName);
```

Comment:
Utils.cleanNameWithAccents calls already Text.escapeIllegalJcrChars (from JCR module) , is it needed to call Text.escapeIllegalJcrChars (gtom ECMS module) here ?
Side note, I don't get why we have two implementations of Text.escapeIllegalJcrChars in  our source code !

Reply:
sorry i wasn't paying attention to the call of the `Text.escapeIllegalJcrChars` method by the `Utils.cleanNameWithAccents` method , i was add just the messing `Utils.cleanNameWithAccents` call .
Call of unnecessary method removed !